fluentd/fluent.conf

```diff
@@ -112,8 +112,10 @@
   tag system.var.log
   format multiline
   format_firstline /^[A-Z][a-z]{2} [ |0-3]+[0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9] /
-  format1 /^(?<time>[^ ]* [^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
+  format1 /^(?<time>[^ ]*[ ]*[^ ]*[ ]*[^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
```

Comment:
On Tue, Nov 17, 2015 at 10:43 AM, Eric Wolinetz <notifications@github.com>
wrote:

> I definitely want + instead of * for the white space... or were you
> thinking more replacing every instance of * with +?
>
I don't really know, that's part of why I wanted a comment with examples :)
Using a * means it could match even if it's not there; using them in
sequence "smells" funny because it's not very specific about what you're
matching. Using a + asserts that you do actually want to make sure it
matches something.

> I also should update line 114 to remove +
>
> I can also update to use \s and \S. I wasn't sure if it would make it
> easier for most to read or not
>
It also has the benefit of considering tabs as spaces, in case anything
uses them.

>